index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```

Comment:
This is good. It lets the player know that they need to refresh to play the game again. A better solution, however, would be to create a dedicated reset button that the player can push to reset the game, eliminating the need to refresh.